src/ILCompiler.ReadyToRun/src/Compiler/DependencyAnalysis/ReadyToRun/PrecodeMethodImport.cs

```diff
@@ -0,0 +1,67 @@
+
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+using Internal.JitInterface;
+using Internal.Text;
+using Internal.TypeSystem;
+
+namespace ILCompiler.DependencyAnalysis.ReadyToRun
+{
+    public class PrecodeMethodImport : Import, IMethodNode
+    {
+        private readonly MethodWithGCInfo _localMethod;
+
+        public PrecodeMethodImport(
+            ReadyToRunCodegenNodeFactory factory,
+            ReadyToRunFixupKind fixupKind,
+            MethodWithToken method,
+            MethodWithGCInfo localMethod,
+            bool isUnboxingStub,
+            bool isInstantiatingStub,
+            SignatureContext signatureContext) :
+            base (
+                factory.PrecodeMethodImports,
+                factory.MethodSignature(
+                      fixupKind,
+                      method,
+                      isUnboxingStub,
+                      isInstantiatingStub,
+                      signatureContext)
+            )
+        {
+            _localMethod = localMethod;
+        }
+
+        public MethodDesc Method => _localMethod.Method;
+
+        public MethodWithGCInfo MethodCodeNode => _localMethod;
+        
+        public override int ClassCode => 30624770;
+
+        public override void EncodeData(ref ObjectDataBuilder dataBuilder, NodeFactory factory, bool relocsOnly)
+        {
+            // This needs to be an empty target pointer since it will be filled in with Module*
```

Comment:
Not your fault but would you mind fixing this imprecise comment to state that the pointer will be filled in with the precode function pointer?